integration_tests/mysql-sink/create_mv.sql

```diff
@@ -31,7 +31,7 @@ CREATE TABLE rw_typed_data (
     boolean_column BOOLEAN,
     date_column DATE,
     time_column TIME,
-    timestamp_column TIMESTAMP,
+    timestamp_column TIMESTAMPTZ,
```

Comment:
It seems the column names are named after types in rw. Is it better to rename to `timestamptz_column`? Do we want to add a `timestamp column` and test its interoperability with mysql `datetime`?

We can fix this test first and make the improvements above as a followup.